### PR TITLE
DOCSP-22047 clarify edit autoload

### PR DESCRIPTION
**Pull request by davemungo**
### STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-22047-clarify-edit-autoload/reference/editor-mode/#editing-a-command

### JIRA
https://jira.mongodb.org/browse/DOCSP-22047
[MONGOSH] Clarify editor autoload

### CODE REVIEW

File: source/reference/editor-mode.txt [hunk 1]
```diff
@@ -211,8 +212,18 @@ it has a syntax error. The highlighted line is missing a comma:
       { "name": "Paola", "group": "marketing" }
    ] )
 
-Enter ``edit`` without any arguments to re-edit the command. The
-external editor reloads the last edit.
+To setup the example:
```
Comment (jmd-mongo) [inline source/reference/editor-mode.txt:215]: [nit]

I think we want "set up" here.

**Review by jmd-mongo — APPROVED** [review]
Thanks for changes, @davemungo. This one LGTM mod a tiny nit.

Thanks!
Joe

File: source/reference/editor-mode.txt (hunk 1 above)
Comment (davemungo) [reply source/reference/editor-mode.txt]: Done

**Review by davemungo — COMMENTED** [review]
Thanks @jmd-mongo  sending to tech review next.

**Comment by davemungo** [comment]
@addaleax Would you have a look at this update when you get a moment? Thanks

**Pull request merged**